docs/theme/build.ts

```diff
@@ -0,0 +1,245 @@
+import StyleDictionary from 'style-dictionary';
```

Comment:
Hah this is super cool, using design tokens and "eating our own dogfood" to create the Monaco themes, I like it :)!